Core/include/Acts/Surfaces/RectangleBounds.hpp

```
@@ -159,8 +159,14 @@ inline double RectangleBounds::get(BoundValues bValue) const {
       return m_min.y();
     case eMaxX:
       return m_max.x();
+    case eMaxY:
+      return m_max.y();
+    case eSize:
+      // eSize case is only necessary to make the switch exhaustive
```

Maybe throw an error here? It is an error, technically.